pyincore/analyses/nonstructbuildingdamage/nonstructbuildingdamage.py

```diff
@@ -33,17 +34,24 @@ def run(self):
         # Building dataset
         building_set = self.get_input_dataset("buildings").get_inventory_reader()
 
+        # building retrofit strategy
+        retrofit_strategy_dataset = self.get_input_dataset("retrofit_strategy")
+
+        # mapping
+        dfr3_mapping_set = self.get_input_dataset("dfr3_mapping_set")
+
+        # Update the building inventory dataset if applicable
+        bldg_dataset, tmpdirname, _ = DatasetUtil.construct_updated_inventories(building_set,
```

Comment:
building_set needs to be a Dataset object, but on line 34 we call get_inventory_reader()

Reply:
Yes I just pushed a fix.